src/mcts/search.cc

```diff
@@ -571,10 +571,10 @@ Eval Search::GetBestEval(Move* move, bool* is_terminal) const {
           best_edge.GetM(parent_m - 1) + 1};
 }
 
-std::pair<Move, Move> Search::GetBestMove() {
+std::pair<Move, Move> Search::GetBestMove(const bool force_temp_to_zero) {
```

Comment:
I think `ignore_temp` is a better name.

Reply:
Thanks for reviewing, I've adapted to your suggestions.